src/core/tranche/assessor.sol

```diff
@@ -47,30 +51,26 @@ contract Assessor is DSNote {
 
     }
 
-    function trancheCount() public returns (uint) {
-        uint count = 0;
-        if (junior != address(0x0)) { count++; }
-        if (senior != address(0x0)) { count++; }
-        return count;
-    }
-
     // --- Calls ---
     function file(bytes32 what, address addr_) public auth {
         if (what == "junior") { junior = addr_; }
         else if (what == "senior") { senior = addr_; }
         else revert();
     }
 
-    function getAssetValue() public returns (uint) {
-         OperatorLike operator = OperatorLike(msg.sender);
-         uint trancheReserve = operator.balance();
-         uint poolValue = pile.Debt();
-         if (msg.sender == junior) {
-            return calcJuniorAssetValue(poolValue, trancheReserve, calcSeniorDebt());
-         }
+    function calcTokenPrice() public returns (uint) {
```

Comment:
maybe removing calc & just naming tokenPrice. same for calcJuniorAssetValue & calcSeniorAssetValue